components/generator/imagevector/src/main/kotlin/io/github/composegears/valkyrie/generator/imagevector/ImageVectorGenerator.kt

```diff
@@ -12,7 +12,7 @@ data class ImageVectorGeneratorConfig(
 
 enum class OutputFormat(val key: String) {
     BackingProperty(key = "backing_property"),
-    LazyDelegateProperty(key = "lazy_delegate_property"),
+    LazyProperty(key = "lazy_property"),
```

Comment:
potentially can break user preference (in our logic it will reset to Backing option)

Reply:
Yeah, that's right.